shared/tests/APIv3.test.ts

```diff
@@ -0,0 +1,143 @@
+import * as APIv4 from "../api/v4";
```

Comment:
You can also move this file to the `tests` folder in backend once you moved the functions.